lib/DebuggerClient.js

```diff
@@ -21,8 +22,9 @@ function createFailingConnection(reason) {
  * @constructor
  * @param {number} debuggerPort
  */
-function DebuggerClient(debuggerPort) {
+function DebuggerClient(debuggerHost, debuggerPort) {
```

Comment:
Can we make this backwards-compatible by changing the order of parameters to `(debuggerPort, debuggerHost)`?